src/terminal/Process.cpp

```diff
@@ -29,7 +29,9 @@
 #include <unordered_map>
 
 #if !defined(_WIN32)
+#if !defined(__FreeBSD__)
```

Comment:
double `#if !...` may be a little weird to look at / review. But no need to change, I can do that later.